cmd/olm/main.go

```diff
@@ -227,5 +234,10 @@ func main() {
 		go monitor.Run(op.Done())
 	}
 
+	// Start the controller manager
+	if err := mgr.Start(ctx.Done()); err != nil {
```

Comment:
this will bypass our clusteroperator reporting, won't it?

Reply:
What part in particular? No reporting that currently exists has been removed, is there anything of the Operator API/controller we want/need to add to reporting at this point?

Reply:
Ah, sorry - I just meant that the healthiness of this control loop does not affect the clusteroperator's health. That is not necessarily something we need to figure out right now